Add prerequisite for AKS import (#1590)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md

6. Click **Save**.

**Result:** The annotation does not give the capabilities to the cluster, but it does indicate to Rancher that the cluster has those capabilities.

## Troubleshooting

This section lists some of the most common errors that may occur when importing a cluster, along with steps to troubleshoot them.

### AKS

#### Error: Getting static credential is not allowed because this cluster is set to disable local accounts.

> * This may occur if your cluster has disabled local accounts. Enable local accounts before attempting to import the cluster again:

Reviewer: ```suggestion
  To resolve this issue, enable local accounts before attempting to [import the cluster](#registering-a-cluster) again.
```

>   ```
>   az aks update --resource-group <resource-group> --name <cluster-name> --enable-local-accounts
>   ```

Reviewer: ```suggestion
    ```sh
    az aks update --resource-group <resource-group> --name <cluster-name> --enable-local-accounts
    ```
```

6. Click **Save**.

**Result:** The annotation does not give the capabilities to the cluster, but it does indicate to Rancher that the cluster has those capabilities.

## Troubleshooting

> This section lists some of the most common errors that may occur when importing a cluster, along with steps to troubleshoot them.

Reviewer 2: ```suggestion
This section lists some of the most common errors that may occur when importing a cluster and provides steps to troubleshoot them.
```
